src/app/components/admin/dashboard/components/site-settings/site-settings.component.html

```diff
@@ -0,0 +1,14 @@
+<form>
+  @for (setting of settings(); track setting) {
+    <baw-range
```

Comment:
not all of these settings will be range inputs. some will be booleans or strings. 

We can deal with that later though